routes/api/stream/index.js

```diff
@@ -26,8 +26,8 @@ router.get('/', (req, res, next) => {
         return asset;
       }),
 
-    // Get the public settings.
-    Setting.getPublicSettings()
+    // Get the moderation setting from the settings.
+    Setting.retrieve()
```

Comment:
This will return all settings including banned words to the client, is that the intention?